src/org/zaproxy/zap/extension/quickstart/QuickStartPanel.java

```diff
@@ -283,6 +299,20 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		return confButton;
 	}
 
+	public void updateConfButton(boolean apiState) {
+		if (confButton == null) {
+			getConfButton();

```

Comment:
This statement can be removed. The button does not need to be updated if it was not initialised nor added to the panel.